src/Network/Network/BYOIP/CustomIpPrefix/UpdateAzureCustomIpPrefixCommand.cs

```diff
@@ -63,11 +63,25 @@ public class UpdateAzureCustomIpPrefixCommand : CustomIpPrefixBaseCmdlet
         [SupportsWildcards]
         public virtual string ResourceId { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
+        [Parameter(Mandatory = false, HelpMessage = "start commissioning process.")]
         public SwitchParameter Commission { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
-        public SwitchParameter Decomission { get; set; }
+        [Alias("Decomission")]
+        [Parameter(Mandatory = false, HelpMessage = "start decommissioning process.")]
+        public SwitchParameter Decommission { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "start provisioning process.")]
+        public SwitchParameter Provision { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "start deprovisioning process.")]
+        public SwitchParameter Deprovision { get; set; }
+
+        [Parameter(
+            Mandatory = true,
```

Comment:
New required parameter will cause breaking change, which is not allowed in regular release, please wait for our major release

Reply:
this is a mistake. just changed to false.